packages/apollo/src/commands/service/__tests__/check.test.ts

```diff
@@ -238,6 +380,130 @@ describe("service:check", () => {
 
   // These are integration tests and not e2e tests because these don't actually hit the remote server.
   describe("integration", () => {
+    describe("federated", () => {
+      describe("should report composition errors correctly", () => {
+        it("vanilla", async () => {
+          captureApplicationOutput();
+          mockCompositionFailure();
+
+          expect.assertions(2);
+
+          await expect(
+            ServiceCheck.run([
+              ...cliKeyParameter,
+              "--serviceName=accounts",
+              `--endpoint=${localURL}/graphql`
+            ])
+          ).rejects.toThrow(
+            federatedServiceCompositionUnsuccessfulErrorMessage
+          );
+
+          // Inline snapshots don't work here due to https://github.com/facebook/jest/issues/6744.
+          expect(uncaptureApplicationOutput()).toMatchSnapshot();
+        });
+
+        it.skip("--markdown", async () => {
```

Comment:
Should this be skipped?

Reply:
Yes it should, I'll leave a comment as to why.